lua/nvim-gps/init.lua

```diff
@@ -92,19 +92,17 @@ local transform_lang = {
 	end
 }
 
+setmetatable(transform_lang, {
+	__index = function()
+		return default_transform
+	end
+})
+
 function M.is_available()
-	return setup_complete and (config.languages[ts_parsers.ft_to_lang(vim.bo.filetype)] == true)
+	return setup_complete and config.languages[ts_parsers.ft_to_lang(vim.bo.filetype)]
 end
 
 function M.setup(user_config)
-	-- By default enable all languages
-	for k, _ in pairs(config.languages) do
```

Comment:
with the `transform_lang` default value set, we don't need to iterate over the languages to set it for the non-set langs.